oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/user/OAuth2User.java

```diff
@@ -55,11 +55,4 @@
 	 */
 	Collection<? extends GrantedAuthority> getAuthorities();
```

Comment:
It's not clear to me why this method would remain. I think `OAuth2AuthenticatedPrincipal` has a `getAuthorities` as well?

Reply:
Yes, I missed that one. PR has ben updated